### ISSUE TITLE
More camera images

### ISSUE THREAD

**Issue by GWCTas** [LMACRC/weather#3]
Is it possible to create a number of images for the day, say 9am, 10, 11, Noon and 1pm and then have them stored where I can display on a page. Then the next day renew all 5?

**Comment by stuartcarnie** [comment]
Yes, we could create and upload one per hour from whenever the day starts to ends.

* webcam_0700.jpg
* webcam_0800.jpg
* …
* webcam_1500.jpg
* webcam_1600.jpg
* webcam_1700.jpg

You probably wouldn't want to display the remaining images from a prior day? e.g.

At 08:30, you would display images for 07:00 and 08:00 captured today, but I don't expect you want to show 09:00 – 17:00 from the prior day?

**Comment by GWCTas** [comment]
Exactly as you put it. No need for historical images.

**Comment by stuartcarnie** [comment]
What will you do for the remaining images after the current time – does it matter if they are from yesterday, or will you only show images up to the current hour?
